tuna/fin_class.py

```diff
@@ -293,18 +292,17 @@ def dump_json(self, outfile, to_file=True):
     if to_file is True:
       if not os.path.exists(outfile):
         os.mknod(outfile)
-      fout = open(outfile, 'w')
-      fout.write("[\n")
-      i = 0
-      while i < len(self.fin_list):
-        json_out = json.dumps(self.fin_list[i])
-        fout.write(json_out)
-        if i != len(self.fin_list) - 1:
-          fout.write(',\n')
-        i += 1
-
-      fout.write("\n]")
-      fout.close()
+      with open(outfile, 'w') as fout: # pylint: disable=unspecified-encoding
```

Comment:
what's the encoding for the json file? I can add it, instead of silencing the warning. (utf-8?)